{
  "_url": "https://github.com/PowerShell/PowerShell/issues/2359",
  "author": "JamesWTruher",
  "body": "## Steps to reproduce\n\nobserve daily results\n## Expected behavior\n\nno errors\n## Actual behavior\n\n [-] Error occurred in Describe block 84ms\n   Exception calling \"Open\" with \"0\" argument(s): \"Connecting to remote server localhost failed with the following error message : Access is denied. For more information, see the about_Remote_Troubleshooting Help topic.\"\n   At C:\\projects\\powershell\\test\\powershell\\Host\\HostUtilities.Tests.ps1:54 char:9\n## Environment data\n\nAppVeyor Daily Build\nhttps://ci.appveyor.com/project/PowerShell/powershell/build/6.0.0-alpha.10-686#L1484\n\n_tplunk has details on how to fix_\n",
  "closed_at": "2016-12-07T21:21:18Z",
  "comments": [
    {
      "author": "PaulHigin",
      "author_association": "COLLABORATOR",
      "body": "I talked with Travis and this break is due to the tests reliance on implicit credentials for a WinRM remote loop-back connection.  This issue has been addressed in PR #2362.\n",
      "created_at": "2016-09-27T16:09:16Z",
      "updated_at": "2016-09-27T16:09:16Z"
    },
    {
      "author": "PaulHigin",
      "author_association": "COLLABORATOR",
      "body": "This issue has been fixed.",
      "created_at": "2016-12-07T21:21:18Z",
      "updated_at": "2016-12-07T21:21:18Z"
    }
  ],
  "created_at": "2016-09-26T19:38:51Z",
  "number": 2359,
  "state": "closed",
  "title": "HostUtilities.Tests.ps1 \"InvokeOnRunspace method on remote runspace\" fails in appveyor (daily build)",
  "updated_at": "2017-03-15T06:51:21Z"
}